third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -1064,6 +1121,17 @@ func resourceDataprocClusterUpdate(d *schema.ResourceData, meta interface{}) err
 		updMask = append(updMask, "config.secondary_worker_config.num_instances")
 	}
 
+<% unless version == 'ga' -%>
+	if d.HasChange("cluster_config.0.lifecycle_config.0.auto_delete_time") {
```

Comment:
Changing `idle_delete_ttl` right now results in a permadiff as you don't submit the change to the API but the API still responds with the original value.